coop_membership/data/email_template_data.xml

```diff
@@ -8,7 +8,47 @@
         <field name="lang">${object.lang}</field>
         <field name="reply_to">${(object.company_id.email or '')|safe}</field>
         <field name="subject">Welcome to Coop</field>
-        <field name="body_html"><![CDATA[]]></field>
+        <field name="body_html"><![CDATA[
+<p>Hello ${object.name},</p>
+
+<p>Your subscription to La Louve has been registered. Welcome !</p>
+
+
+% if object.shift_type == 'standard':
+<p>Your member number is $ {object.barcode_base} and you have been registered on the niche ${object.current_template_name}.</p>
+% endif
+
+% if object.shift_type == 'ftop':
+<p>Your membership number is $ {object.barcode_base} and you have been enrolled in the steering wheel team. You will find attached a document which explains the functioning of this team as well as a calendar which indicates the settlement dates.</p>
+% endif
+
+<p>You will receive a reminder a few days before your next service. <br/>
+You can now come and discover the store and do your shopping, upon presentation of an identity document.</p>
+
+<p>We invite you to connect to <a href="https://membres.cooplalouve.fr/"> The Members Area of ​​La Louve </a> (in attachment, the procedure to activate it). You will find there a lot of information, in particular your next services, and a forum which will allow you to communicate between members. The current version of the site will evolve and new functionalities will be added in the coming months. A team of volunteer members works to enrich this space to make it useful and convivial.</p>
```

Comment:
https://membres.cooplalouve.fr should be replaced by something else.. probably odoo url

and "The Members Area of La Louve" to "The Members Area"

Reply:
@ivantodorovich : I used company website